keras_nlp/models/roberta.py

```diff
@@ -170,6 +171,184 @@ def get_config(self):
         return config
 
 
+class RobertaMultiSegmentPacker(keras.layers.Layer):
```

Comment:
Is this in roberta because the same exact padding and packing will get used for Roberta just with the bpe tokenizer?